src/esp/gfx/PbrTextureUnit.h

```diff
@@ -9,21 +9,30 @@
 #include <Corrade/Containers/EnumSet.h>
 
 namespace esp {
+typedef unsigned char uint8_t;
```

Comment:
:scream: why? 

This is a standard type, `#include <cstdint>`.

Reply:
I was getting a warning and I thought this was better than an include :D

Reply:
I don't think aliasing `uint8_t` / `std::uint8_t` with `esp::uint8_t` is a good idea.